src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static Ball getRandomBall() {
```

Comment:
Consider changing the method from static to a non-static method, as it would be more appropriate for an object-oriented design.